rfcs/python_38.md

```diff
@@ -0,0 +1,42 @@
+# RFC 186: Bump minimum supported Python to 3.8
```

Comment:
```suggestion
# RFC 189: Bump minimum supported Python to 3.8
```

Reply:
Ah, I forgot to update it. Did it now. Thanks!